core/src/main/java/org/mule/transformer/simple/AddFlowVariableTransformer.java

```diff
@@ -6,20 +6,27 @@
  */
 package org.mule.transformer.simple;
 
-import org.mule.PropertyScope;
+import org.mule.api.MuleEvent;
+import org.mule.api.metadata.DataType;
+import org.mule.transformer.types.TypedValue;
 
 public class AddFlowVariableTransformer extends AbstractAddVariablePropertyTransformer
 {
 
-    @Override
-    protected PropertyScope getScope()
-    {
-        return PropertyScope.INVOCATION;
-    }
-    
     public void setVariableName(String variableName)
```

Comment:
This is used by spring i imagine, ideally should be using an alias rather than adding this methods..